Added front for the ToDo

ThomasCandille opened the pull request:
Front end seulement avec les affichages pour le front, a voir avec quentin comment faire l'intégration du backend

ThomasCandille committed:
✨ First commit to my branch (added react)

ThomasCandille committed:
🔥 Changed all for ToDo (only front)

Kan-A-Pesh commented on frontend/README.md:
Plz change the README or delete it

Kan-A-Pesh reviewed (changes requested):
ltgm (without the README)

ThomasCandille committed:
💄 Refactor docker-compose.yml and frontend files

- Removed unnecessary comments and unused services in docker-compose.yml
- Updated frontend README.md with project overview and installation instructions
- Added .gitignore file to ignore .env file
- Added docker-compose.dev.yml, .dockerignore, and Dockerfile to support development and production stages in frontend
-merci michel <3

Kan-A-Pesh committed:
Merge branch 'main' into feat-front
